.github/workflows/mac.yml

```
@@ -16,398 +16,330 @@ concurrency:
   cancel-in-progress: true
 
 env:
+  MACOSX_DEPLOYMENT_TARGET: '11.0'
   PYTHON_VERSION: '3.10'
-  OV_BRANCH: master
-  OV_TARBALL: ''
+  TARGET_BRANCH: ${{ github.base_ref || github.event.merge_group.base_ref || github.ref }}
 
 jobs:
   openvino_download:
-    name: Download OpenVINO package
+    name: Download OpenVINO
     outputs:
       status: ${{ steps.openvino_download.outcome }}
+      ov_artifact_name: ${{ steps.openvino_download.outputs.ov_artifact_name }}
+      ov_wheel_source: ${{ steps.openvino_download.outputs.ov_wheel_source }}
     timeout-minutes: 10
     defaults:
       run:
         shell: bash
-    runs-on: ubuntu-22.04
+    runs-on: aks-linux-2-cores-8gb
+    container:
+      image: 'openvinogithubactions.azurecr.io/openvino_provider:0.1.0'
+      volumes:
+        - /mount:/mount
+        - ${{ github.workspace }}:${{ github.workspace }}
+
     steps:
-      - name: Download OpenVINO build
-        id: openvino_download
-        run: |
-          wget ${{ env.OV_TARBALL}} --progress=bar:force:noscroll -O openvino_package.tar.gz
-          tar -tvf openvino_package.tar.gz
-        continue-on-error: true
+    - uses: openvinotoolkit/openvino/.github/actions/openvino_provider@master
+      id: openvino_download
+      with:
+        platform: macos_12_6
+        commit_packages_to_provide: wheels
+        revision: latest_nightly
```

```suggestion
        revision: nightly
```
such naming looks better IMO, as nightly is always assumed as latest

but understand that it should be fixed in a separate repo (if more people agree)